src/Config/upiconfig.xml

```diff
@@ -8,13 +8,13 @@
         <upi_attribute name='id' value='DYN'/>
         <upi_element name='level'>
           <upi_attribute name='name' value='release'/>
-          <upi_attribute name='id' value='3.6.0'/>
+          <upi_attribute name='id' value='3.6.1'/>
```

Comment:
Dont think this need to be changed, please refer to previous PRs. I think in UPI terms, release and build are two different things